src/app/component/eco-news/components/news-list/news-list.component.ts

```diff
@@ -56,8 +58,16 @@ export class NewsListComponent implements OnInit, OnDestroy {
   }
 
   public onResize(): void {
+    this.getSessionStorageView();
     this.windowSize = window.innerWidth;
-    this.view = (this.windowSize >= 576) ? this.view : true;
+    this.view = this.windowSize < Breakpoints.tabletLow ? true : (this.gallery ? true : false);
+  }
+
+  private getSessionStorageView() {
+    const view = sessionStorage.getItem('viewGallery');
+    if (view !== null) {
```

Comment:
you can check it like this: if(view)....

Reply:
I think, "if (view !== null)" is corectly, because "if(view)" will work only when view=true. But i need value "true" and also "false"